src/wolfgame.py

```diff
@@ -6131,11 +6131,11 @@ def admin_endwho(event, var, target):
 
     channels.Main.who()
 
-@cmd("coin", pm=True)
-def coin(cli, nick, chan, rest):
+@command("coin", pm=True)
+def coin(var, wrapper, message):
     """It's a bad idea to base any decisions on this command."""
 
-    reply(cli, nick, chan, messages["coin_toss"].format(nick))
+    wrapper.send(messages["coin_toss"].format(wrapper.source.nick))
```

Comment:
In `str.format` calls, you can pass in a context (e.g. `wrapper.source`) directly and it will work as intended. We try to do that everywhere that it makes sense, as we try to move away from nick-based operations :)